exonum/src/encoding/protobuf/mod.rs

```diff
@@ -126,11 +126,9 @@ impl ProtobufConvert for DateTime<Utc> {
     }
 
     fn from_pb(pb: well_known_types::Timestamp) -> Result<Self, ()> {
-        if pb.get_nanos() >= 0 {
-            Ok(Utc.timestamp(pb.get_seconds(), pb.get_nanos() as u32))
-        } else {
-            Err(())
-        }
+        Utc.timestamp_opt(pb.get_seconds(), pb.get_nanos() as u32)
```

Comment:
Please remind me, is it well-defined to cast a negative `i32` to `u32` in Rust? I _guess_ it should be fine and we should not change this. But maybe some day they'll insert a sign check in debug builds.